.buildkite/pipeline-release-helm.yml

```diff
@@ -0,0 +1,41 @@
+steps:
+  - group: helm-release
+    steps:
+
```

Comment:
We probably should do similar to the operatorhub release now and build this binary once, and distribute it as an artifact to the other steps: https://github.com/elastic/cloud-on-k8s/pull/6377/files#diff-368ac783357785c9197a32bec71fd23db2530688d2316db6e23a358d76960bbbL13

Reply:
Done in a3a24ab.